### PR TITLE
Add support for setting the broadcasted name on Android

### PR DESCRIPTION
**Pull request by marvinmarnold**

### CODE REVIEW

**Comment by marvinmarnold** [comment]
Also added ability to turn on BT discovery.

File: README.md [hunk 1]
```diff
@@ -609,13 +611,52 @@ See [discoverUnpaired](#discoverunpaired).
...
+Not currently implemented.
+
+### Parameters
+
+- __discoverableDuration__: Desired time device should be discoverable for.
 
 ### Quick Example
 
-    bluetoothSerial.clearDeviceDiscoveredListener();
+    bluetoothSerial.setName(0);
```
Comment (evanliomain) [inline README.md]: You should mean `bluetoothSerial.setDiscoverable(0);` rather than `bluetoothSerial.setName(0);`.

Comment (marvinmarnold) [inline README.md]: Thanks, fixed in 7f84998

**Comment by don** [comment]
@marvinmarnold thanks for adding these features.

Your changes nuked the documentation for [clearDeviceDiscoveredListener](https://github.com/don/BluetoothSerial/pull/137/files#diff-04c6e90faac2675aa89e2176d2eec7d8L611), can you re-add that?

**Comment by don** [comment]
@marvinmarnold sorry it took so long to merge...

**Comment by marvinmarnold** [comment]
@don np, tx

**Pull request merged**